common/web/gesture-recognizer/src/includes/events.ts

```diff
@@ -2,19 +2,142 @@
 // browser-compatible way.  (Just requires a blank 'module' object on the window.)
 ///<reference path="../../../../../node_modules/eventemitter3/index.js" />
 
-// Unfortunately, I can't get it to recognize type information properly
-// because we can't use require statements.  So, a small-scale manual definition.
-declare class EventEmitter {
-  /** Add a listener for a given event */
-  on(event: string, func: (...args: any[]) => void, context?: any);
-  /** Add a one-time listener for a given event */
-  once(event: string, func: (...args: any[]) => void, context?: any);
-  removeListener(event: string, func: (...args: any[]) => void, context?: any, once?: boolean);
-
-  // Defines their alternately-themed aliases.
-  addListener: typeof EventEmitter.prototype.on;
-  off: typeof EventEmitter.prototype.removeListener;
-
-  // Defines the actual event-raising function.
-  emit(eventName: string, ...args: any[]);
+// A full copy of the module's actual .d.ts file, save for the one line that prevents us from
+// actually referencing it.  (Since the project doesn't use modules.)
```

Comment:
If we prefer, I _could_, in theory, add something to the build script to nix that line, or to replicate the original file and then nix it for use instead of having this hardcoded copy.

The big benefit this gives us:  solid Intellisense & typing for any class's supported events.  With the old version, we didn't really get typing or suggestions about supported events when using the related functions here.